product_taxes_group/model/tax_group.py

```diff
@@ -0,0 +1,213 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Product - Taxes Group module for Odoo
+#    Copyright (C) 2014 -Today GRAP (http://www.grap.coop)
+#    @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import logging
+
+from openerp.osv import fields, osv
+from openerp.osv.orm import Model
+from openerp import SUPERUSER_ID
+from openerp.tools.translate import _
+
+_logger = logging.getLogger(__name__)
+
+
+class tax_group(Model):
```

Comment:
@legalsylvain You should put your model metadata (_columns, _defaults) at the begining of your class definition.
Regards